mlir/test/Integration/GPU/LEVELZERO/gpu-reluf32-to-spirv.mlir

```diff
@@ -0,0 +1,79 @@
+// RUN: mlir-opt %s -pass-pipeline='builtin.module(spirv-attach-target{ver=v1.0 caps=Addresses,Int64,Kernel},convert-gpu-to-spirv{use-64bit-index=true},gpu.module(spirv.module(spirv-lower-abi-attrs,spirv-update-vce)),func.func(llvm-request-c-wrappers),convert-scf-to-cf,convert-cf-to-llvm,convert-arith-to-llvm,convert-math-to-llvm,convert-func-to-llvm,gpu-to-llvm{use-bare-pointers-for-kernels=true},gpu-module-to-binary,expand-strided-metadata,lower-affine,finalize-memref-to-llvm,reconcile-unrealized-casts)' \
```

Comment:
Can you use `convert-to-llvm`? That should be the modern way to perform LLVM conversion:

```suggestion
// RUN: mlir-opt %s -pass-pipeline='builtin.module(spirv-attach-target{ver=v1.0 caps=Addresses,Int64,Kernel},convert-gpu-to-spirv{use-64bit-index=true},gpu.module(spirv.module(spirv-lower-abi-attrs,spirv-update-vce)),func.func(llvm-request-c-wrappers),convert-scf-to-cf,convert-to-llvm,gpu-module-to-binary,expand-strided-metadata,lower-affine,finalize-memref-to-llvm,reconcile-unrealized-casts)' \
```

I see that there is an option for `gpu-to-llvm{use-bare-pointers-for-kernels=true}` so that may have to be done ahead of time separately, but we should look into using the dlti infra to replace this in the future I think (@fabianmcg : is this on your radar maybe?)

Reply:
Thanks, @joker-eph. Using `-convert-to-llvm` for all `to-llvm` conversions except `-gpu-to-llvm`.
I think we should be able to remove it once @fabianmcg 's changes land.